webhook-bridge/svix-webhook-bridge.example5.receivers.yaml

```diff
@@ -0,0 +1,50 @@
+log_level: "debug"
+log_format: "json"
+
+# Optionally configure these settings to export OTEL data
+opentelemetry:
+  address: "http://localhost:1234"
+  service_name: "my-bridge"
+  sample_ratio: 0.5
+
+# Host/port to listen on for HTTP inputs.
+# Only required when HTTP receivers are specified, probably?
+http_listen_address: "0.0.0.0:5000"
+
+
+# Each defines an `input`, (optional) `transformation`, and `output`.
+receivers:
+  - name: "from-svix-to-queue"
+    input:
+      # The "webhook" input type uses the HTTP listener. POST bodies/headers are "handled" by forwarding to the
+      # configured "output."
+      type: "webhook"
```

Comment:
I prefer "webhook" to "http" since we're biased towards POST requests with JSON objects for bodies. I'd expect an "HTTP" input to have a lot more config options to let me go wild, but "webhook" sort of curbs that feeling for me.